src/main/webapp/i18n/de/team.json

```
@@ -183,7 +183,7 @@
                 "question": "Sollen alle Teams der Übung <strong>{{ title }}</strong> und ihre Teilnahmen an der Aufgabe wirklich dauerhaft gelöscht werden? Diese Aktion kann NICHT rückgängig gemacht werden!",
                 "typeNameToConfirm": "Bitte gib den Titel der Übung zur Bestätigung ein."
             },
-            "importError": "Import fehlgeschlagen. Bitte versuche es nochmal oder kontaktiere einen Administrator für Hilfe.",
+            "importError": "Import fehlgeschlagen. Bitte versuche es nochmal oder kontaktiere die Adminstration für Hilfe.",
```

```suggestion
            "importError": "Import fehlgeschlagen. Bitte versuche es erneut oder kontaktiere die Adminstration für Hilfe.",
```
sounds a bit more formal